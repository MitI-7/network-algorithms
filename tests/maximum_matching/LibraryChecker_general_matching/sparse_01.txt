500 198 113
364 143
167 257
282 447
233 274
330 259
204 434
196 142
1 324
430 223
276 419
340 265
80 356
368 451
453 91
423 174
218 285
369 162
374 103
214 224
131 260
25 67
219 158
409 196
354 423
371 399
405 40
6 477
236 345
35 107
215 47
159 66
218 122
113 91
471 384
19 214
52 3
312 443
213 94
466 163
224 24
6 150
474 221
436 444
10 384
269 101
212 381
94 93
371 432
178 174
285 11
436 458
360 181
231 455
27 23
195 487
466 31
489 298
414 382
10 104
378 300
261 117
104 162
402 141
173 386
164 260
491 107
118 14
188 385
139 365
10 26
122 133
474 341
52 483
158 189
296 432
350 430
488 55
499 92
477 491
400 142
341 459
442 430
347 4
499 471
86 484
54 259
470 483
301 263
252 240
402 185
156 480
187 334
163 456
299 340
340 335
367 129
92 7
148 115
44 302
87 368
38 278
284 95
32 148
120 373
81 335
192 391
3 58
172 361
313 495
375 236
79 380
16 462
152 13
72 313
367 375
326 152
289 274
396 350
106 357
125 168
179 172
437 306
328 440
322 331
122 224
371 20
432 108
488 239
320 303
308 204
95 476
69 413
150 45
382 379
396 354
445 360
204 251
262 150
395 290
459 245
498 441
21 152
207 261
106 366
365 222
347 269
100 85
14 1
319 467
9 184
206 415
218 149
4 251
142 293
357 184
471 413
184 441
387 455
395 97
108 157
168 61
371 126
193 6
289 251
394 149
473 458
256 77
425 162
394 193
269 368
365 416
437 287
343 250
160 250
484 267
42 142
251 140
468 27
182 482
328 45
179 178
150 110
254 323
398 493
77 244
57 193
126 470
297 372
258 274
87 226
447 141
363 15
315 262
445 235
128 227
317 184
156 171
206 367