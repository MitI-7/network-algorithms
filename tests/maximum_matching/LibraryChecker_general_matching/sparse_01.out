113
1 324
3 58
4 347
6 477
7 92
9 184
10 26
11 285
13 152
14 118
15 363
16 462
19 214
20 371
23 27
24 224
25 67
31 466
32 148
35 107
38 278
40 405
42 142
44 302
45 328
47 215
52 483
54 259
55 488
57 193
61 168
66 159
69 413
72 313
77 256
79 380
80 356
81 335
85 100
86 484
87 226
91 453
93 94
95 284
97 395
101 269
103 374
104 162
106 357
108 157
110 150
117 261
120 373
122 133
126 470
128 227
129 367
131 260
139 365
140 251
141 447
143 364
149 218
156 480
158 219
160 250
163 456
167 257
172 361
173 386
174 423
178 179
181 360
182 482
185 402
187 334
188 385
192 391
195 487
196 409
204 434
206 415
212 381
221 474
223 430
231 455
233 274
235 445
236 345
240 252
245 459
254 323
262 315
263 301
265 340
276 419
287 437
296 432
297 372
298 489
300 378
303 320
312 443
319 467
322 331
350 396
368 451
379 382
384 471
398 493
436 444
441 498
458 473
